src/main/groovy/jenkins/automation/utils/ScmUtils.groovy

```diff
@@ -24,8 +32,9 @@ class ScmUtils {
                         branch("*/tags/$version")
                     }
                     relativeTargetDir(repo.sub_directory)
+                    shallowClone(shallow)
```

Comment:
Initialize it to false (default) above, but I think when it is null it is defaulted to true
http://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.helpers.scm.GitContext.shallowClone